README.md

```diff
@@ -0,0 +1,62 @@
+
+# Upload your sections and articles to Zendesk  
+
+From a FileTree with folders and HTML articles, upload them to your Zendesk Help Center.  
+ 
+## **Goal**
+  
+The goal of this plugin is to provide a way to upload a whole FileTree containing sections & articles to Zendesk.  
+  
+## **How to use it**
+
+### **Gradle**
+Add the following to your `build.gradle`:
+```groovy
+buildscript {
+  repositories {
+    maven {
+      url "https://plugins.gradle.org/m2/"
+    }
+  }
+  dependencies {
+    classpath "com.saagie:htmltozendeskuploader:0.3.0"
+  }
+}
+
+apply plugin: "com.saagie.htmltozendeskuploader"
+```
+Then you can use the task HtmlToZendeskTask (with the list of parameters below).
+
+### **Registering the task in Kotlin Script**
+```kotlin
+register<HtmlToZendeskTask>("THE_NAME_YOU_WANT_TO_GIVE") {
+  group = TASK_GROUP
+  sourceDir = SOURCE_DIR  
+  targetCategoryId = ZENDESK_CATEGORY_ROOT  
+  apiBasePath = ZENDESK_API_BASE_PATH  
+  user = ZENDESK_USERNAME  
+  password = ZENDESK_PASSWORD  
+  pattern = VERSION_PATTERN 
+}
+```
+
+### **Parameters of the task**
+- **group: String**
+Gradle group task in which will be the task.
+- **sourceDir: InputDirectory**
+Source directory containing the sections and the HTML files representing the articles. The source directory will be the section root in Zendesk, directly under the category root.
+- **targetCategoryId: Long**
+Zendesk category ID in which will be uploaded all the sections & articles.
+- **apiBasePath: String**
+Your Zendesk API endpoint.
+- **user: String**
+The username used to push the articles.
```

Comment:
pour harmoniser le username & le password, tu peux mettre : `the username of the Zendesk account`